src/tox_travis/hooks.py

```diff
@@ -11,13 +11,12 @@ def tox_addoption(parser):
         '--travis-after', dest='travis_after', action='store_true',
         help='Exit successfully after all Travis jobs complete successfully.')
 
-    default_toxenv()
-
 
 @tox.hookimpl
 def tox_configure(config):
     """Check for the presence of the added options."""
     if config.option.travis_after:
-        travis_after_monkeypatch()
+        travis_after_monkeypatch(config)
 
     override_ignore_outcome(config)
+    default_toxenv(config)
```

Comment:
If it works to have it here, I love it. When I wrote the initial implementation, I figured this would be too late. If it's not, this is great.

We need to check to make sure that we have tests covering uses of `toxenv -e`, as well as any default `TOXENV`, to make sure that we _don't_ override those. Those are escape hatches when all else fails.

If we end up that we're going to use the `config` everywhere, then I think we should just go ahead and make `override_ignore_outcome` be called from `default_toxenv`. It was only ever split because it needed to happen in a different hook.

Extra thought: should the hooks just be implemented in the `toxenv` and `after` modules? Pluggy lets there be lots of hook implementations, so it seems like it would be reasonable, and would keep them more separated.